examples/Simple Vertically Partitioned SplitNN.ipynb

```diff
@@ -226,26 +227,27 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Epoch 0 - Training loss: 0.5366069674491882\n",
-      "Epoch 1 - Training loss: 0.2596180737018585\n",
-      "Epoch 2 - Training loss: 0.19612883031368256\n",
-      "Epoch 3 - Training loss: 0.1600290834903717\n",
-      "Epoch 4 - Training loss: 0.13417688012123108\n"
+      "Epoch 0 - Training loss should be: \"running_loss/len(dataloader)\"\n",
```

Comment:
We can give the dataloader a `__len__` method in a different PR, but for this PR we can hardcode the number of batches, which for 128 batch size would be 469